packages/types/src/namada.ts

```diff
@@ -1,9 +1,10 @@
-import { SupportedTx } from "@namada/types";
 import { AccountType, DerivedAccount } from "./account";
 import { Signer } from "./signer";
 
 export type TxMsgProps = {
-  txType: SupportedTx;
+  //TODO: figure out if we can make it better
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  txType: any;
```

Comment:
Maybe this could just be specified as a `string` here, as `TxType` is really just a string from an enum. We can leave this as is for now though, just in case it causes any errors elsewhere!

Reply:
I will check it out!